renderer/components/image-drop.js

```diff
@@ -0,0 +1,57 @@
+//  validate file format with a list of format supports imgur
+let validateFile = (file) => {
+  let imageFormat = ['jpeg', 'png', 'gif', 'peg', 'apng', 'tiff', 'pdf', 'xcf']
```

Comment:
Use const, only use let if you realize you need to change this variable later (tip: never do it, use immutable code)